erpnext/hr/utils.py

```diff
@@ -363,6 +354,34 @@ def check_frequency_hit(from_date, to_date, frequency):
 			return True
 	return False
 
+def check_frequency_hit(from_date, to_date, frequency, based_on_date_of_joining_date):
```

Comment:
Rename the function and delete the old above function

Reply:
done